### DISCUSSION TITLE
image with transparent background isn't fully transparent

### DISCUSSION THREAD

**Discussion by rtwfroody** [tailwindlabs/tailwindcss#12273] [Help] [answered]
Instead, it looks like the transparency makes the page background lighter. See https://mealinventor.com/ to see what I'm talking about.

I'm sure I'm missing something completely obvious.

**Comment by wongjn** [comment, marked as answer]
It seems like the image background isn't fully transparent. It seems to be translucent: https://play.tailwindcss.com/9isx1xaxwY

**Reply by rtwfroody** [reply]
Ugh. I should have seen that. Thank you!
